app/Controllers/Event.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace App\Controllers;
+
+use App\Controllers\BaseController;
+use App\Models\EventModel;
+use App\Models\SpeakerModel;
+
+class Event extends BaseController
+{
+    public function get(int|null $year = null) {
+        if ($year === null) {
+            // todo: fetch the currently displayed year from the database
+            $year = 2024;
```

Comment:
should we agree on a current year?
I use 2025.
Or is it just not important?

Reply:
I don't think that it's important right know. In the end, the frontend should never assume a default year, but instead use what the backend defaults to. Especially during development, I'd recommend defaulting to 2024, because we want to "rebuild" the page of 2024. Is that okay for you?

Reply:
That is okay for me. :)